wagtail-{{ cookiecutter.project_name_kebab }}/.circleci/config.yml

```diff
@@ -7,27 +7,27 @@ jobs:
     steps:
       - checkout
       - run: pip install flake8
-      - run: flake8 wagtail_{{ cookiecutter.project_name_snake }}
+      - run: flake8 wagtail_localize
 
   prettier:
     docker:
-      - image: circleci/node:14
+      - image: circleci/node:16
     steps:
       - checkout:
 
       - type: cache-restore
         keys:
-          - node-modules-{% raw %}{{ .Branch }}-{{ checksum "package-lock.json" }}{% endraw %}
-          - node-modules-{% raw %}{{ .Branch }}-{% endraw %}
+          - node-modules-{{ .Branch }}-{{ checksum "package-lock.json" }}
```

Comment:
I believe we need the ``{% raw %}`` here as we don't want cookiecutter to parse the ``{{ }}`` syntax (we want this to be parsed by Circle CI instead)